docs/reference/install-faq.rst

```diff
@@ -158,3 +158,28 @@ or
   python3 /opt/rocm-6.2.0/libexec/rocm_smi/rocm_smi.py
 
 See `Symbolic links in wheels <https://discuss.python.org/t/symbolic-links-in-wheels/1945>`_ for more information.
+
+.. _troubleshooting-denylist:
+
+Issue #8: The AMDGPU driver is not loaded after installation
+======================================================================================
+
+Ensure the AMDGPU driver is not on a denylist such as ``/etc/modprobe.d/blacklist-amdgpu.conf``.
+The location of this file might vary depending on the system distribution and version.

```

Comment:
The rest of the page is structured as:
* An issue/problem, labeled with a number
* **Solution:** provided.  

Are you able to structure it in a similar way, so the page is more consistent?  

As a follow up task, if you dislike the "Issue/solution" format for this troubleshooting guide, then, feel free to restructure the page to a uniform format.